### PR TITLE
Feature/io 751 GitHub actions ci cd

### PR DESCRIPTION
**Pull request by nikolay-ahramovich-idf**

### CODE REVIEW

**Review by ekaterina-demeshko-idf — APPROVED** [review]

**Review by andrei-lastochkin-idf — APPROVED** [review]

File: .github/workflows/build_commit.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          SECRETS_FILE_CONTENT: ${{ secrets.SECRETS_FILE_CONTENT }}
+      - run: flutter pub get
+      - run: echo $GOOGLE_SERVICES_FILE_CONTENT | base64 -d > android/app/google-services.json
+        env:
+          GOOGLE_SERVICES_FILE_CONTENT: ${{ secrets.GOOGLE_SERVICES_FILE_CONTENT }}
+      - name: Build APK
+        run: flutter build apk --flavor prod -t lib/main_prod.dart --release
+      - name: Create Github Release
+        id: create-new-release
+        uses: actions/create-release@v1
```
Comment (sergei-mikhailovskii-idf) [inline .github/workflows/build_commit.yaml]: image

let's use the maintained repository

File: .github/workflows/build_commit.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        env:
+          GITHUB_TOKEN: ${{secrets.TOKEN_GITHUB}}
+        with:
+          tag_name: v${{github.run_number}}
+          release_name: Release V${{github.run_number}}
+      - name: Upload asset to GitHub Release
+        uses: actions/upload-release-asset@v1
```
Comment (sergei-mikhailovskii-idf) [inline .github/workflows/build_commit.yaml]: -//-

File: android/app/build.gradle [hunk 3]
```diff
@@ -49,7 +49,7 @@ android {
         applicationId "com.example.star_movie_app"
         // You can update the following values to match your application needs.
         // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-build-configuration.
-        minSdkVersion flutter.minSdkVersion
+        minSdkVersion localProperties.getProperty('flutter.minSdkVersion').toInteger()
```
Comment (sergei-mikhailovskii-idf) [inline android/app/build.gradle]: why did u change it?

**Review by sergei-mikhailovskii-idf — CHANGES_REQUESTED** [review]
rename the files please, specify the build type in the name, like

build_prod_on_master_commit
build_sandbox_on_master_pr

or smth similar

File: android/app/build.gradle (hunk 3 above)
Comment (nikolay-ahramovich-idf) [reply android/app/build.gradle]: Because min 19 sdk version is required, didn't want to hardcode

File: .github/workflows/build_commit.yaml (hunk 1 above)
Comment (nikolay-ahramovich-idf) [reply .github/workflows/build_commit.yaml]: Fixed.

File: .github/workflows/build_commit.yaml (hunk 2 above)
Comment (nikolay-ahramovich-idf) [reply .github/workflows/build_commit.yaml]: Fixed.

**Review by sergei-mikhailovskii-idf — APPROVED** [review]

**Pull request merged**